scripts/checkpatch.sh

```diff
@@ -50,7 +50,23 @@ case "$op" in
 		;;
 	*)
 		echo "Checking commit(s):"
-		for c in $*; do checkpatch $c; done
+    version="$(git --version | awk -F '.' '{print $2}')"
+    if (( version < 19 )); then
```

Comment:
This only checks the minor version. What if the major version is increased?

Reply:
updated to check major version too